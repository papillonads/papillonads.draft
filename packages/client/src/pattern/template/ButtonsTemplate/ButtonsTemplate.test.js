import React from 'react'
import buttonsTemplateDefault from './ButtonsTemplate.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<ButtonsTemplate />', () => {
  describe('Snaphot', () => {
    test('must match buttonsTemplateDefault', () => {
      expect(global.renderToJSON(buttonsTemplateDefault)).toMatchSnapshot()
    })
  })
})
