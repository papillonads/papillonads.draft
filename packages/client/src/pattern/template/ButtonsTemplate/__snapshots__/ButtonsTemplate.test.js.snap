// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonsTemplate /> Snaphot must match buttonsTemplateDefault 1`] = `
<div>
  <h2 />
  <div>
    Under construction
     
    <span
      aria-label="emoji"
      role="img"
    >
      ðŸ™‚
    </span>
    <div />
  </div>
</div>
`;
